engine/src/main/java/org/terasology/rendering/assets/shader/ShaderProgramFeature.java

```diff
@@ -21,11 +21,11 @@
 /**
  */
 public enum ShaderProgramFeature {
-    FEATURE_REFRACTIVE_PASS (0b00000001),
-    FEATURE_ALPHA_REJECT (0b00000010),
-    FEATURE_LIGHT_POINT (0b00000100),
-    FEATURE_LIGHT_DIRECTIONAL (0b00001000),
-    FEATURE_USE_MATRIX_STACK (0b00100000),
+    FEATURE_REFRACTIVE_PASS      (0b00000001),
+    FEATURE_ALPHA_REJECT         (0b00000010),
+    FEATURE_LIGHT_POINT          (0b00000100),
+    FEATURE_LIGHT_DIRECTIONAL    (0b00001000),
+    FEATURE_USE_MATRIX_STACK     (0b00100000),
```

Comment:
I knew it. I knew this would be a can of worms. =P

I recommend to revert the change then, leaving no spaces and no alignment. Not as pretty but I'd rather not create a new precedent where CheckStyle is ignored.